app/components/post_header/post_header.js

```diff
@@ -124,11 +125,14 @@ export default class PostHeader extends PureComponent {
             commentedOnDisplayName,
             commentCount,
             isBot,
+            isLandscape,
         } = this.props;
 
         const style = getStyleSheet(theme);
         const showReply = shouldRenderReplyButton || (!commentedOnDisplayName && commentCount > 0 && renderReplies);
-        const displayNameStyle = [style.displayNameContainer, showReply && (isBot || fromAutoResponder || fromWebHook) ? style.displayNameContainerBotReplyWidth : null];
+        const reduceWidth = showReply && (isBot || fromAutoResponder || fromWebHook);
+        const isLandscapeStyle = isLandscape && reduceWidth ? style.displayNameContainerLandscapeBotReplyWidth : isLandscape ? style.displayNameContainerLandscape : null; //eslint-disable-line no-nested-ternary
```

Comment:
Too complex

Reply:
This has been updated to be easier to understand